docs/quick_start.md

```diff
@@ -231,27 +231,17 @@ The Jstz dev wallet supports only the Chrome web browser.
 
 :::
 
-1. Download the source code for the dev wallet:
+1. Download the [latest release](https://github.com/jstz-dev/dev-wallet/releases/latest) of the wallet and unpack it.
 
-   ```bash
-   git clone https://github.com/jstz-dev/dev-wallet.git
-   ```
-
-1. Go into the repository, install the dependencies, and build the wallet:
-
-   ```bash
-   cd dev-wallet && pnpm i && pnpm build
-   ```
-
-1. In Chrome, open the extensions page at `chrome://extensions`.
+2. In Chrome, open the extensions page at `chrome://extensions`.
 
-1. At the top right of the page, use the radio button to enable **Developer mode**.
+3. At the top right of the page, use the radio button to enable **Developer mode**.
```

Comment:
It isn't necessary to renumber these steps; if you put every numbered step as 1, the markdown processor will automatically number them in the output. Not big deal, if you'd rather have the correct numbers in the source.